scipy/io/matlab/_miobase.py

```diff
@@ -79,7 +79,7 @@ class MatReadWarning(UserWarning):
          '''unicode_strings : bool, optional
    If True, write strings as Unicode, else MATLAB usual encoding.'''}
 
-docfiller = doccer.filldoc(doc_dict)
+docfiller: doccer.Decorator = doccer.filldoc(doc_dict)
```

Comment:
The type argument is missing here. 
But it's not needed to use `doccer.Decorator` here; static type-checkers don't require it. 
So you could just write 

```suggestion
docfiller: Final = doccer.filldoc(doc_dict)
```

(assuming that `typing.Final` is imported)